lib/options.json

```
@@ -283,10 +283,17 @@
       "link": "https://webpack.js.org/configuration/dev-server/#devserverhistoryapifallback"
     },
     "Host": {
-      "type": "string",
-      "minLength": 1,
       "description": "Allows to specify a hostname to use.",
-      "link": "https://webpack.js.org/configuration/dev-server/#devserverhost"
+      "link": "https://webpack.js.org/configuration/dev-server/#devserverhost",
+      "anyOf": [
+        {
+          "enum": ["local-ip", "local-ipv4", "local-ipv6"]
+        },
+        {
+          "type": "string",
+          "minLength": 1
+        }
+      ]
```

`schema-utils` should respect `parentSchema.link/description`, if we don't have link/description for children.

```suggestion
      "anyOf": [
        {
          "enum": ["local-ip", "local-ipv4", "local-ipv6"],
          "link": "https://webpack.js.org/configuration/dev-server/#devserverhost"
        },
        {
          "type": "string",
          "minLength": 1,
          "link": "https://webpack.js.org/configuration/dev-server/#devserverhost"
        }
      ]
```